app/assets/javascripts/home.coffee

```diff
@@ -106,7 +106,7 @@ set_error_for_invalid_fields = ->
       $('#submission_provider_down_speed').removeClass('got-error')
       $('#speed_error_span').addClass('hide')
 
-window.addEventListener('DOMContentLoaded', ->
+$ ->
```

Comment:
This is neat because all our assets are stored locally and don't need to be requested through an time-consuming request. Load time for our assets will be near instant.